rules-tests/DowngradePhp70/Rector/GroupUse/SplitGroupedUseImportsRector/Fixture/status_controller_update_command.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+    namespace App\Command;
+
+    use App\Entity\Order;
+    use Symfony\Component\Console\{Command\Command, Style\SymfonyStyle, Input\InputInterface, Output\OutputInterface};
+    use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+    class StatusControllerUpdateCommand
+    {
+    }
+?>
+-----
+<?php
+
+    namespace App\Command;
+
+    use App\Entity\Order;
+    use Symfony\Component\Console\Command\Command;
+    use Symfony\Component\Console\Style\SymfonyStyle;
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Output\OutputInterface;
```

Comment:
@TomasVotruba The `use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;` is missing here, that's why the failure show 2 lines different

Reply:
@samsonasik  I see. Thanks :+1:

Reply:
@xenon87 Could you update the fixture with expected invalid code?

Reply:
Looks good now, thanks :+1: